contributed_definitions/NXem.nxdl.xml

```diff
@@ -0,0 +1,723 @@
+         Both have electro-magnetic lenses. These lens may differ in design, alignment,
+         number, and level of corrected-for aberrations. As an obvious difference,
+         a TEM is used mainly to measure the transmitted electron beam.
+         This demands thinner specimens as in SEM but offers capabilities for probing 
+         of additional physical mechanisms of electron-matter interaction.
+         
+         Compared to SEMs, TEMs have a different relative arrangement between the
+         lenses and the specimen which is most obvious by the different relative
+         arrangement of the objective lens versus the specimen. Nevertheless, both 
+         types of electron microscopes use detector systems, which measure
+         different types of signals that originate though from the same set of
+         radiation/specimen interactions. Consequently, detectors can also be similar.
+         
+         Given these physical and technical differences, different instruments have
+         been developed. This lead to a coexistence of two broad interacting
+         communities: SEM and TEM users. From a data science perspective, we 
+         acknowledge that the more specific a research question is and the narrower
+         the addressed user base which develops or uses schemata for
+         research data management with EM data and metadata, the more understandable 
+         it is that scientists of either community (or sub-community) ask for
+         method-specific schemata.
+         
+         Researchers who have a single (main) microscope of some vendor in their lab,
+         may argue they need an NXem_vendor_name schema or an NXem_microscope_name or
+         an NXem_sem or a NXem_tem schema. Scientists exclusively working with one
+         technique or type of signal probed (X-rays, electrons) may argue they wish
+         to be pragmatic and store only what is immediately relevant for their
+         particular technique and research questions. In effect, they may advocate
+         for method-specific schemata such as NXem_ebsd, NXem_eels, 
+         NXem_edx, or NXem_imaging.
+         
+         The development in the past has shown that these activities have led to a
+         zoo of schemata and implementations of these into data and file formats.
+         There is nothing which prevents the communities to make these schemata
+         open and interoperable. Open here means specifically not that all data are
+         compliant with/or use the schema and have to end up in the open-source domain.
+         There can be embargo periods first of all. Open means that the metadata and 
+         associated schemata are documented in a manner that as many details as 
+         possible are open in the sense that others can understand what the 
+         (meta)data mean conceptually. 
+         The [FAIR](https://doi.org/10.1038/sdata.2016.18) principles guide all 
```

Comment:
markdown-style reference needs to be converted to reST-style

Reply:
fixed